tests/backup/backup_license_test.go

```diff
@@ -0,0 +1,201 @@
+			log.InfoD("Total number of worker nodes in source and destination cluster are %v", len(totalNumberOfWorkerNodes))
+			log.InfoD("Switching cluster context back to source cluster")
+			err := SetSourceKubeConfig()
+			log.FailOnError(err, "Switching context to source cluster")
+		})
+		Step("Verifying the license count after adding source and destination clusters", func() {
+			log.InfoD("Verifying the license count after adding source and destination clusters")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			licenseInspectResponse, err = Inst().Backup.InspectLicense(ctx, licenseInspectRequestObject)
+			log.FailOnError(err, "Getting the license for organisation %v", orgID)
+			licenseResponseInfoFeatureInfo = licenseInspectResponse.GetLicenseRespInfo().GetFeatureInfo()
+			dash.VerifyFatal(licenseResponseInfoFeatureInfo[0].Consumed, int64(len(totalNumberOfWorkerNodes)), fmt.Sprintf("Verifying the license count when source cluster with %d worker nodes and destination cluster with %d worker nodes are added to backup", len(sourceClusterWorkerNodes), len(destinationClusterWorkerNodes)))
+		})
+		Step("Applying label portworx.io/nobackup=true to worker nodes and verifying the license count", func() {
+			log.InfoD("Applying label portworx.io/nobackup=true to one of the worker node on source cluster and verifying the license count")
+			err := Inst().S.AddLabelOnNode(sourceClusterWorkerNodes[0], "portworx.io/nobackup", "true")
+			if err == nil {
+				sourceClusterWorkerNodeLabelApplied = true
+				sourceClusterWorkerNodeLabelRemoved = false
+			}
+			log.FailOnError(err, fmt.Sprintf("Failed to apply label portworx.io/nobackup=true to worker node %v", sourceClusterWorkerNodes[0].Name))
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			licenseCountUpdateCheck := func() (interface{}, bool, error) {
+				licenseInspectResponse, err = Inst().Backup.InspectLicense(ctx, licenseInspectRequestObject)
+				if err != nil {
+					return "", false, err
+				}
+				licenseResponseInfoFeatureInfo = licenseInspectResponse.GetLicenseRespInfo().GetFeatureInfo()
+				if licenseResponseInfoFeatureInfo[0].Consumed == int64(len(totalNumberOfWorkerNodes)-1) {
+					return "", false, nil
+				}
+				return "", true, nil
+			}
+			_, err = task.DoRetryWithTimeout(licenseCountUpdateCheck, licenseCountUpdateTimeout, licenseCountUpdateRetryTime)
+			licenseInspectResponse, err = Inst().Backup.InspectLicense(ctx, licenseInspectRequestObject)
+			log.InfoD("License count after applying label portworx.io/nobackup=true to node %v is %v ", sourceClusterWorkerNodes[0].Name, licenseInspectResponse.GetLicenseRespInfo().GetFeatureInfo()[0].Consumed)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying license count after applying label portworx.io/nobackup=true to node %v", sourceClusterWorkerNodes[0].Name))
```

Comment:
1)Handle the error returned, before doing the logging in Inst().Backup.InspectLicense(ctx, licenseInspectRequestObject),
2)My main intension of including 'InspectLicense' after DoRetryWithTimeout is to know the license count in case of error